### PR TITLE
Use process.cwd() instead of __dirname for baseDir

### PR DESCRIPTION
**Pull request by dgeb**
cc @rwjblue

### CODE REVIEW

**Comment by dgeb** [comment]
I've gotta get some more tests in this repo ...

**Comment by rwjblue** [comment]
What happened with __dirname?

**Comment by dgeb** [comment]
> What happened with __dirname?

When building projects (like glimmer-di) I was seeing errors (unfortunately masked by heimdalljs):

```
dgeb-mbp:glimmer-di dgeb$ npm run build

> @glimmer/di@0.1.10 build /Users/dgeb/src/glimmerjs/glimmer-di
> rm -rf dist && BROCCOLI_ENV=dist broccoli build dist

Build project: @glimmer/di
Build env: dist
Build path: /Users/dgeb/src/glimmerjs/glimmer-di
Build tsconfig:  tsconfig.json
TypeError: cannot stop: not the current node
    at Cookie.stop (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:1625:15)
    at /Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:1948:23
    at /Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:1367:34
    at tryCatch (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:533:12)
    at invokeCallback (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:545:13)
    at publish (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:516:7)
    at publishRejection (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:459:3)
    at flush (/Users/dgeb/src/glimmerjs/glimmer-build/node_modules/heimdalljs/dist/heimdalljs.cjs.js:1525:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

Build failed
```

File: lib/to-es5.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function ImportExternalHelpersPlugin() {
   };
 }
 
-ImportExternalHelpersPlugin.baseDir = function() { return __dirname; };
+ImportExternalHelpersPlugin.baseDir = function() { return process.cwd(); };
```
Comment (stefanpenner) [inline lib/to-es5.js]: this isn't correct, we will be scanning and include unexpected things in the cache key. Disable the persistent cache instead, until the root cause is understood.

**Pull request merged**